package.json

```diff
@@ -14,7 +14,7 @@
     "@types/lodash": "^4.14.179",
     "@types/react": "latest",
     "@types/react-dom": "latest",
-    "aptos": "^1.3.7",
+    "aptos": "^1.3.14",
     "aptos-node-checker-client": "0.0.2",
     "date-fns": "^2.29.1",
     "graphql": "^16.6.0",
```

Comment:
shouldn't `typescript` and `prettier` be dev dependencies?

Reply:
yes it in line 35 and line 24 if that's what you mean?